app/javascript/angular/code/filtersUtils.js

```diff
@@ -79,15 +79,17 @@ export const setupClipboard = () => {
   new Clipboard("#copy-link-button");
 };
 
-export const tooltipInstance = tooltipId =>
+export const tooltipInstance = (tooltipId, oldTooltip) =>
   tippy(`#${tooltipId}`, {
     animateFill: false,
     interactive: true,
     trigger: "manual"
-  })[0];
+  })[0] || oldTooltip;
+
+let tooltip;
```

Comment:
I don't like this `tooltip` hanging around. An alternative is to use the [module pattern](https://coryrylan.com/blog/javascript-module-pattern-basics) like

```
const tooltipInstance = (() => {
  let instance;

  return tooltipId => {
    if (!instance) {
      instance = tippy(`#${tooltipId}`, {
        animateFill: false,
        interactive: true,
        trigger: "manual"
      })[0];
    }
    return instance;
  }
})();
```

Reply:
Awesome!